service/discovery/azure/compute.go

```diff
@@ -321,7 +321,7 @@ func (d *azureComputeDiscovery) discoverSlots(baseSite *armappservice.Site, c ar
 		s    voc.IsCompute
 	)
 
-	pager := d.clients.sitesClient.NewListSlotsPager(util.Deref(d.rg), util.Deref(baseSite.Name), &armappservice.WebAppsClientListSlotsOptions{})
+	pager := d.clients.sitesClient.NewListSlotsPager(resourceGroupName(*baseSite.ID), util.Deref(baseSite.Name), &armappservice.WebAppsClientListSlotsOptions{})
```

Comment:
util.Deref(baseSite.ID)` would be safer against nil pointer exceptions although the ID shouldn't be empty ever :)